# HG changeset patch
# User Kevin McCarthy <kevin@8t8.us>
# Date 1514495241 28800
#      Thu Dec 28 13:07:21 2017 -0800
# Node ID 2874b82116df6b5d92bd305354d8ea1ad4ad9c37
# Parent  a67340471339f947a278aedeff8af70c7e5248e9
Mention self_encrypt options in the documentation. (closes #3983)

Add a paragraph to the "Sending Cryptographically Signed/Encrypted
Messages" section of the manual.

diff -r a67340471339 -r 2874b82116df doc/manual.xml.head
--- a/doc/manual.xml.head	Sun Dec 17 13:36:21 2017 +0100
+++ b/doc/manual.xml.head	Thu Dec 28 13:07:21 2017 -0800
@@ -1476,6 +1476,14 @@ be encrypted using the selected public k
 </para>
 
 <para>
+To ensure you can view encrypted message you have sent, you
+may wish to set <link linkend="pgp-self-encrypt">$pgp_self_encrypt</link>
+and <link linkend="pgp-self-encrypt-as">$pgp_self_encrypt_as</link> for PGP, or
+<link linkend="smime-self-encrypt">$smime_self_encrypt</link>
+and <link linkend="smime-self-encrypt-as">$smime_self_encrypt_as</link> for S/MIME.
+</para>
+
+<para>
 Most fields of the entries in the key selection menu (see also <link
 linkend="pgp-entry-format">$pgp_entry_format</link>) have obvious
 meanings.  But some explanations on the capabilities, flags, and
